### PR TITLE
Change term predictedColumns to predictedActiveColumns in GeneralTemporalMemory

### PR DESCRIPTION
**Pull request by chetan51**
Fixes https://github.com/numenta/nupic.research/issues/278.

### CODE REVIEW

**Comment by chetan51** [comment]
@scottpurdy

File: sequence_prediction/sequence_generator/predict.py [hunk 1]
```diff
@@ -324,7 +325,7 @@ def accuracy(self):
   for numPredictions in NUM_PREDICTIONS:
     runners.append(Runner(numPredictions))
 
-  for i in xrange(100000000):
+  for i in iter(int, 1):
```
Comment (scottpurdy) [inline sequence_prediction/sequence_generator/predict.py]: what?

File: sequence_prediction/sequence_generator/predict.py [hunk 2]
```diff
@@ -295,7 +296,7 @@ def step(self):
 
 
     # Feed noise
-    sequence = range(NUM_SYMBOLS, NUM_SYMBOLS + RANDOM_RESERVOIR)
+    sequence = range(RANDOM_START, RANDOM_END)
```
Comment (scottpurdy) [inline sequence_prediction/sequence_generator/predict.py]: xrange?

**Comment by scottpurdy** [comment]
:+1:

File: sequence_prediction/sequence_generator/predict.py (hunk 1 above)
Comment (chetan51) [inline sequence_prediction/sequence_generator/predict.py]: Oh oops, I accidentally included other unrelated changes into this PR. Will clean up before merging.

Comment (scottpurdy) [inline sequence_prediction/sequence_generator/predict.py]: :angry:

Comment (chetan51) [inline sequence_prediction/sequence_generator/predict.py]: It's okay though cause it's `nupic.research`! :smiling_imp:
